parm/metplus_config/prep/cam/chem_grid2obs/ASCII2NC_obsAIRNOW.conf

```diff
@@ -0,0 +1,57 @@
+[config]
+
+OUTPUT_BASE = {ENV[DATA]}
+
+PROCESS_LIST = ASCII2NC
+LOOP_ORDER = processes
+LOOP_BY = VALID
+
```

Comment:
@Ho-ChunHuang-NOAA We still have an instance of LOOP_ORDER in a configuration file.  Would you please remove it?